src/ServiceManagement/Services/Commands.Utilities/Websites/WebsitesClient.cs

```diff
@@ -1360,9 +1360,43 @@ private DeploymentBaseOptions CreateRemoteDeploymentBaseOptions(string websiteNa
                 AuthenticationType = "Basic",
                 TempAgent = false
             };
+            
+            var azureSessionUserAgent = GetUserAgentForAzureSession();
+            if (!string.IsNullOrEmpty(azureSessionUserAgent))
+            {
+                remoteBaseOptions.UserAgent = azureSessionUserAgent;
+            }
 
             return remoteBaseOptions;
         }
+        
+        /// <summary>
+        /// Gets user agent for azure session
+        /// </summary>
+        /// <returns>useragent string </returns>
+        private string GetUserAgentForAzureSession()
```

Comment:
This is only going to get the powershell cmdlet version, and not the version of the management client.  You can get the management client user agent from the HttpClient property of the WebsitesManageemntClient.